src/brain/issueLabelHandler/followups.ts

```diff
@@ -2,50 +2,49 @@ import { EmitterWebhookEvent } from '@octokit/webhooks';
 import * as Sentry from '@sentry/node';
 import moment from 'moment-timezone';
 
-import { ClientType } from '@/api/github/clientType';
 import {
-  RESPONSE_DUE_DATE_FIELD_ID,
+  GH_APPS,
   SENTRY_MONOREPOS,
   SENTRY_REPOS,
-  STATUS_FIELD_ID,
   WAITING_FOR_COMMUNITY_LABEL,
   WAITING_FOR_LABEL_PREFIX,
   WAITING_FOR_PRODUCT_OWNER_LABEL,
   WAITING_FOR_SUPPORT_LABEL,
 } from '@/config';
+import { ClientType } from '@api/github/clientType';
+import { getClient } from '@api/github/getClient';
+import {
+  calculateSLOViolationRoute,
+  calculateSLOViolationTriage,
+} from '@utils/businessHours';
 import {
   addIssueToGlobalIssuesProject,
   isNotFromAnExternalOrGTMUser,
   modifyDueByDate,
   modifyProjectIssueField,
   shouldSkip,
-} from '@/utils/githubEventHelpers';
-import { getClient } from '@api/github/getClient';
-import {
-  calculateSLOViolationRoute,
-  calculateSLOViolationTriage,
-} from '@utils/businessHours';
+} from '@utils/githubEventHelpers';
 import { isFromABot } from '@utils/isFromABot';
 
 const REPOS_TO_TRACK_FOR_FOLLOWUPS = new Set([
   ...SENTRY_REPOS,
   ...SENTRY_MONOREPOS,
 ]);
 
-function isNotInARepoWeCareAboutForFollowups(payload) {
+function isNotInARepoWeCareAboutForFollowups(app, payload) {
```

Comment:
This is a lot of the change here, adding `app` to all of the `shouldSkip`s, because a few of them need it. Now that I think about it, it might've been better to make that the second arg, that way if it's not needed it doesn't matter. 😬 😭

Reply:
agreed here, optional parameter seems to make sense instead of passing unused ones.

Reply:
Done in 1793a82662a507af391475f947c5386ed76ecc0d.